docs/S01-fundamentals/M2-consensus/L5-trustless-consensus/index.html

```diff
@@ -94,7 +93,11 @@ <h3>Periods and Proof of Work</h3>
       </ul>
 
       <p>
-        The following section will describe the process of creating a block and how blocks chained together create an immutable ledger that all network participants can validate on their own. This individual validation, without having to rely on trust, is how proof of work consensus maintains state in a distributed and safe way. 
+<<<<<<< HEAD
+        The following section will describe the process of creating a block and how blocks chained together create an immutable ledger that all network participants can validate on their own. This individual validation, without having to rely on trust, is how proof of work consensus maintains state in a distributed and safe way.
```

Comment:
Hey! I think this is an issue with the conflict resolution?

Reply:
removed conflict market